templates/registration/password_reset_complete.html

```diff
@@ -0,0 +1,9 @@
+<!-- templates/registration/password_reset_complete.html -->
+{% extends 'base.html' %}
+
+{% block title %}Password reset complete{% endblock %}
+
+{% block content %}
+<h1>Password reset complete</h1>
+<p>Your new password has been set. You can log in now on the <a href="{% url 'login' %}">log in page</a>.</p>
```

Comment:
Suggestion:
You can log in now on the <a href="{% url 'login' %}">log in page</a>
>
You can <a href="{% url 'login' %}"> log in </a> now.